public/js/constants.js

```diff
@@ -938,6 +952,7 @@
         exports.VARIANT.AMBIGUOUS,
         exports.VARIANT.BLUERED,
         exports.VARIANT.ACIDTRIP,
+	exports.VARIANT.MULTISINGLERAINBOW,
```

Comment:
Please make separate commits for separate changes.

Reply:
Yeah, Zamiel asked me to just submit it despite it not being tested or reviewed. He said he would fix it up.